unix-cheat-sheet.tex

```diff
@@ -150,9 +150,9 @@ \subsection*{Input output redirection}
                                             & output of \cv{cmd1} to the input of \cv{cmd2}\\
 \shcmd{\cv{cmd}}{> file}                    & runs \cv{cmd} and redirects output to \cv{file},\\
                                             & content of \cv{file} is overwritten\\
-\shcmd{\cv{cmd}}{>\null> file}              & runs \cv{cmd} and redirects output to \cv{file},\\
-                                            & the output is appended to \cv{file}\\
-\shcmd{\cv{cmd}}{< file}                    & runs \cv{cmd} and redirects \cv{file} to its input
+\shcmd{\cv{cmd}}{>\null> file}              & like \shcmd{}{>>} but appended output to \cv{file}\\
+\shcmd{\cv{cmd}}{< file}                    & runs \cv{cmd} and redirects \cv{file} to its input\\
```

Comment:
I'd write "appends" instead of "appended" here.

Reply:
Make that "append" and I agree